subscription-manager.spec

```diff
@@ -116,6 +116,13 @@ from the server. Populates /etc/docker/certs.d appropriately.
 %{_sysconfdir}/rhsm/pluginconf.d/container_content.ContainerContentPlugin.conf
 %{rhsm_plugins_dir}/container_content.py*
 %{_datadir}/rhsm/subscription_manager/plugin/container.py*
+# Copying Red Hat CA cert into each directory:
+%dir %{_sysconfdir}/docker/certs.d/cdn.redhat.com
+%dir %{_sysconfdir}/docker/certs.d/access.redhat.com
+%dir %{_sysconfdir}/docker/certs.d/registry.access.redhat.com
+%{_sysconfdir}/docker/certs.d/cdn.redhat.com
+%{_sysconfdir}/docker/certs.d/access.redhat.com
+%{_sysconfdir}/docker/certs.d/registry.access.redhat.com
```

Comment:
python-rhsm sets the file and dir attrs specifically, ala:
%attr(755,root,root) %dir %{_sysconfdir}/rhsm/ca

%{python_sitearch}/rhsm/*
%{python_sitearch}/rhsm-*.egg-info
%attr(644,root,root) %{_sysconfdir}/rhsm/ca/*.pem

Suspect subman should as well.